tests_generated/test_deployment_language-typescript/.copier-answers.yml

```diff
@@ -14,6 +14,7 @@ indexer_token: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
 project_name: test_deployment_language-typescript
 python_linter: ruff
 use_github_actions: true
+use_pre_commit: <commit>
```

Comment:
Looks like the tests will need updating, they are currently replacing the `true` value with `<commit>`. This is will be due to this regex here https://github.com/algorandfoundation/algokit-beaker-default-template/blob/main/tests/test_templates.py#L107. Just need to add a `^` to match start of line